static/app/views/issueDetails/streamline/activitySection.spec.tsx

```diff
@@ -111,7 +111,7 @@ describe('StreamlinedActivitySection', function () {
     expect(await screen.findByText('Test Note 1')).toBeInTheDocument();
     expect(await screen.findByText('Test Note 7')).toBeInTheDocument();
     expect(screen.queryByText('Test Note 6')).not.toBeInTheDocument();
-    expect(await screen.findByText('4 comments hidden')).toBeInTheDocument();
+    expect(await screen.findByText('4 activities hidden')).toBeInTheDocument();
```

Comment:
comments means something else in activity - this seems the most accurate, but happy to change if anyone has a better idea